roles/cfg_openwrt/tasks/conditional_packages.yml

```diff
@@ -12,6 +12,19 @@
     - airos_dfs_reset|default([])|length > 0
     - role == 'corerouter'
 
+- name: "Add Dawn + Luci app for corerouters"
+  set_fact:
+    packages: "{{ packages + [item] }}"
+  when:
+    - role == 'corerouter' or role == 'ap'
```

Comment:
Doesn't this line basically make sure that all routers + access points in a site will have dawn enabled (if it's been set to enabled)?

Reply:
I thought only, when bandsteering is enabled in the host? (see the 3 lines to the bottom).

Reply:
The Corerouter  & the APs?